Ladybird/WebContentView.cpp

```diff
@@ -576,7 +576,7 @@ void WebContentView::create_client()
 
         auto webcontent_fd_passing_socket_string = DeprecatedString::number(wc_fd_passing_fd);
 
-        Vector<StringView> arguments {
+        Vector<StringView, 5> arguments {
```

Comment:
Why?

Reply:
There's a maximum of 5 arguments here, so I thought making it a little stack array would be nice (it's not an important thing).

Reply:
That's a bit textbook premature IMO, but anyways, in the future a change like that should be in its own commit. It had nothing to due with the stated commit message / PR as a whole.